controller/array_action/array_mediator_ds8k.py

```diff
@@ -566,14 +565,14 @@ def _create_flashcopy(self, source_volume_id, target_volume_id, options):
     def _delete_target_volume_if_exist(self, target_volume_id):
         self._delete_volume(target_volume_id, not_exist_err=False)
 
-    def _create_snapshot(self, target_volume_name, pool_id, source_volume_name):
-        source_volume = self._get_api_volume_by_name(source_volume_name, pool_id=pool_id)
+    def _create_snapshot(self, target_volume_name, pool_id, source_volume_id):
+        source_volume = self._get_api_volume_by_id(source_volume_id)
```

Comment:
please move this line to `create_snapshot`, so the `source_volume_name` parameter of `_generate_snapshot_response` won't have `volume_id`

Reply:
done